### ISSUE TITLE
LenB() returns wrong result for KEYBDINPUT

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#136]
**Describe the bug**
The LenB() for the KEYBDINPUT returns in VB6 the value 28, but in tB it returns 32.
Thus the API does not work.

**Edit**: Len() returns 28 in both tB and VB6.

**To Reproduce**
```
Private Type KEYBDINPUT
dwType As Long
wVKey As Integer
wScan As Integer
dwFlags As Long
Time As Long
dwExtraInfo As Long
dwPadding As Currency
End Type

Dim KEYBDI As KEYBDINPUT
MsgBox LenB(KEYBDI)
```

**Expected behavior**
That tB returns also 28. I assume it is some padding issue.

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.9.2720]

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting.  I'll get that sorted in the next update.  Might be a couple of days as I'm just in the middle of a bigger update👍

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.313, thanks for reporting
